<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inicio_acceso</class>
 <widget class="QMainWindow" name="inicio_acceso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1324</width>
    <height>667</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>inicio_acceso</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget1">
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <widget class="QRadioButton" name="preg1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="preg2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>420</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="preg3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="preg4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>520</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="good">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>230</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; color:#ff0000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1361</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Imagenes/inicio/PANDEMIA.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="inicio">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>600</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#d92905;&quot;&gt;iniciar&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color: rgb(255, 63, 15);
background-color: rgb(230, 36, 22);
background-image: url(:/Imagenes/inicio/fondo-coronavirus-verde-realista_52683-36592.jpg);</string>
    </property>
    <property name="text">
     <string>iniciar</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>preg1</zorder>
   <zorder>preg2</zorder>
   <zorder>preg3</zorder>
   <zorder>preg4</zorder>
   <zorder>good</zorder>
   <zorder>inicio</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1324</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
